### ISSUE TITLE
SUPERB Leaderboard Artefacts

### ISSUE THREAD

**Issue by Hguimaraes** [s3prl/s3prl#383]
Hi guys,

Thank you for this amazing repository and all the work that has been done!
I was looking at issue #268 and saw that some models were fine-tuned w.r.t the learning rate parameter.

Do you think it is possible to also host the CKPT files for the downstream models from the other models? (e.g. WavLM, LightHuBERT, DistillHuBERT, Robust wav2vec2.0, Robust HuBERT mgr...). If not the CKPT, at least the hyperparameters used to reproduce the values on the Leaderboard.

Context: I have been working with the robustness and compression of SRL models. I have trained some downstream tasks, but some of them take a very long time to train (almost a week) and it is necessary to fine-tune in some cases yet.

**Comment by leo19941227** [comment]
Hello!

Sorry that for the models you mentioned, they are not finetuned by us.
The researchers benchmarked their models with S3PRL and submitted to the leaderboard.
Unfortunately, our submission files do not contain the finetuning hyperparameters.
However, I know that WavLM authors have detailed their finetuning hypers in one of our issue page, perhaps you can look into that.
For the others, I think the only way might be to contact the paper authors.

Sincerely,
Leo

**Comment by Hguimaraes** [comment]
Hi Leo,

Thank you very much for your answer! I will try to contact them, and if I get lucky enough to reproduce their results, I can send you the artifacts to be included in the repository, if you want (-:

I saw that HuBERT and wav2vec2.0 are listed here: http://140.112.21.28:8000/
Do you have any other models that may be uploaded there?

And by the way, I tried to reproduce the KS for the Wav2vec2.0 (Base and Large)... I have close results, but not the expected:
- wav2vec2.0 Base: from your ckpt = 96.20 / at the leaderboard = 96.23
- wav2vec2.0 Large: from your ckpt = 96.27 / at the leaderboard = 96.66

I don't know if this is the expected behavior... Could you confirm if this difference is expected?

Best,
Heitor

**Comment by leo19941227** [comment]
I think the different is expected. The difference is not so significant in my opinion.
